### PR TITLE
WEBUI-424: fix full text input requiring enter key press( Backport 10.10)

### PR DESCRIPTION
**Pull request by GuruprasadAnk1996**
https://jira.nuxeo.com/browse/WEBUI-424

### CODE REVIEW

**Review by swarnadipa-dev — DISMISSED** [review]
Looks good

**Review by SnehalNaik-DEV — DISMISSED** [review]
Looks good

**Review by rajatgoel-gl — DISMISSED** [review]

**Review by rakeshkumar1019 — DISMISSED** [review]
looks good 👍

**Review by SnehalNaik-DEV — DISMISSED** [review]
Looks good.

**Review by Gabez0r — COMMENTED** [review]
Please see comments in https://github.com/nuxeo/nuxeo-web-ui/pull/1607#pullrequestreview-1147101310.

File: elements/search/trash/nuxeo-trash-search-form.html [hunk 1]
```diff
@@ -39,7 +39,7 @@
                  value="{{searchTerm}}"
                  placeholder="[[i18n('defaultSearch.fullText.placeholder')]]"
                  autofocus
-                 on-keydown="_checkForEnter">
+                 on-change="_checkForEnter">
```
Comment (rajatgoel-gl) [inline elements/search/trash/nuxeo-trash-search-form.html]: Gabriel Review comments are not implemented 
WEBUI424

**Review by rajatgoel-gl — CHANGES_REQUESTED** [review]

**Review by poonamyadav252 — COMMENTED** [review]
please check your `open issues`.

**Review by rakeshkumar1019 — APPROVED** [review]
looks good.

**Review by swarnadipa-dev — APPROVED** [review]
Looks good

**Review by Nishant0928 — APPROVED** [review]

**Review by Gabez0r — APPROVED** [review]

**Review by rajatgoel-gl — APPROVED** [review]

**Pull request merged**